tests/backends/storage/test_fs.py

```diff
@@ -0,0 +1,107 @@
+"""Tests for Ralph fs storage backend"""
+
+from collections.abc import Iterable
+from pathlib import Path
+
+import pytest
+
+from ralph.backends.storage.fs import FSStorage
+from ralph.defaults import APP_DIR, FS_STORAGE_DEFAULT_PATH
+
+
+# pylint: disable=invalid-name
+# pylint: disable=unused-argument
+def test_fs_storage_instanciation(fs):
+    """Test the FSStorage backend instanciation"""
+    # pylint: disable=protected-access
+
+    assert FSStorage.name == "fs"
+
+    storage = FSStorage()
+
+    assert storage._path == Path(FS_STORAGE_DEFAULT_PATH)
+
+    if Path("multi").exists():
+        assert False
+
+    multi_path = "multi/directories/path"
+
+    storage = FSStorage(multi_path)
+
+    assert storage._path == Path(multi_path)
+
+
+# pylint: disable=invalid-name
+# pylint: disable=unused-argument
+def test_fs_getfile(fs):
+    """Test that a path that exists can be returned, and throws an exception otherwise"""
+    # pylint: disable=protected-access
+
+    path = "test_fs/"
+    filename = "some_file"
+    storage = FSStorage(path)
+
+    with pytest.raises(FileNotFoundError):
+        storage._get_filepath(filename)
+
+    open(path + filename, "w").close()
```

Comment:
Use the `fs` fixture for that:

```suggestion
    fs.create_file(Path(path, filename))
```

Reply:
Fixed